MTM-50209 Added brief introduction to extensions (#977)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/managing-applications.md

</tbody>
</table>

<a name="extensions"></a>
### Extensions

> Click **Extensions** in the **Ecosystem** menu in the navigator to display a list or grid of all extensions in your account.

Reviewer: ```suggestion
Click **Extensions** in the **Ecosystem** menu in the navigator to display all extensions in your account.
```

> Extensions provide better shareability and reusability of UI features across different applications. UI features can be developed as plugins and added to an application without coding knowledge. There are two types of extensions:

Reviewer 2: ```suggestion
Extensions make it easier to share and reuse UI features across different applications. UI features can be developed as plugins and added to an application without coding knowledge. There are two types of extensions:
```

> Extensions provide better shareability and reusability of UI features across the different applications. UI features can be developed as plugins and added to an application without coding knowledge. There are two types of extensions:

Reviewer: ```suggestion
Extensions provide better shareability and reusability of UI features across different applications. UI features can be developed as plugins and added to an application without coding knowledge. There are two types of extensions:
```

- Plugins
- Blueprints

> Blueprint applications can be installed, while plugins are added to other applications. This allows you to scaffold entire solutions or to extend existing ones. Due to the micro frontend technology, this can happen at runtime without rebuilding.

Author: ```suggestion
Blueprint applications must be deployed, while plugins are added to other applications. This allows you to scaffold entire solutions or to extend existing ones. Due to the micro frontend technology, this can happen at runtime without rebuilding.
```

> Blueprint applications can be installed, while plugins can be added into any other application. This allows you to scaffold whole solutions or to extend existing ones. Due to the micro frontend technology, this can happen at runtime and no re-building is needed.

Reviewer 2: ```suggestion
Blueprint applications can be installed, while plugins are added to other applications. This allows you to scaffold entire solutions or to extend existing ones. Due to the micro frontend technology, this can happen at runtime without rebuilding.
```

> Blueprint applications can be installed, while plugins can be added into any other application. This allows you to scaffold whole solutions or to extend existing ones. Due to the micro frontend technology, this can happen at runtime and no re-building is needed.

Reviewer 2: i'm not sure if "can be installed" is the right choice (for blueprint applications). maybe "must be installed"? or is there another way of making them work?

Author: Changed to "must be deployed"

> Blueprint applications can be installed, while plugins can be added into any other application. This gives the ability to scaffold whole solutions or to extend existing ones.  Due to the micro frontend technology, this can happen at run-time and no re-building is needed.

Reviewer: ```suggestion
Blueprint applications can be installed, while plugins can be added into any other application. This allows you to scaffold whole solutions or to extend existing ones. Due to the micro frontend technology, this can happen at runtime and no re-building is needed.
```
